pkgs/development/libraries/gpgme/default.nix

```diff
@@ -9,11 +9,13 @@ let
   gpgProgram = if useGnupg1 then "gpg" else "gpg2";
 in
 stdenv.mkDerivation rec {
-  name = "gpgme-1.8.0";
+  name = "gpgme-1.7.1";
```

Comment:
I think it'd be better to add a `gpgme_1_7` package instead of downgrading the main version; there are other users of gpgme in the tree after all :)